scripts/cli/util/util.py

```diff
@@ -0,0 +1,148 @@
+import copy
+import os
+import socket
+import subprocess
+import time
+
+from os.path import join
+
+def set_environ():
+  '''This function inserts the necessary environment variables to the os.environ
+  dictionary and returns the old one so that can be restored later.'''
+  old = copy.copy(os.environ)
+
+  cc = os.environ['CODECOMPASS_HOME']
+
+  os.environ['SASL_PATH'] = join(cc, 'lib', 'sasl2')
+  # os.environ['MAGIC'] = join(cc, 'share', 'misc', 'magic.mgc')
+
+  ld_lib = join(cc, 'lib') + ':' + join(cc, 'lib64')
```

Comment:
The value of `LD_LIBRARY_PATH` environment variable is more complex. In case of parsing it should contain the library of parser plugins, and in case of starting the web server it should contain the library of server plugins. In case of parsing the `CLASSPATH` should also be set to contain the third party `.jars`.